js/pages/samples.js

```diff
@@ -1,19 +1,19 @@
-import { getMyData, hasUserData } from "../shared.js";
+import { getMyData, hasUserData, translateHTML, translateTex } from "../shared.js";
```

Comment:
I believe there might be a small typo. It seems like you meant to use `translateText` instead of `translateTex`.